Unified/actor.py

```diff
@@ -197,6 +197,9 @@ def singleRecovery(url, task, initial, actions, do=False):
                         if split['splitAlgo'] in ['EventBased']:
                             sendLog('actor',"Changing the splitting on %s for %s is not permitted. Not changing."%(split['splitAlgo'],initial["RequestName"]), level='critical')
                             continue
+                        if split['splitAlgo'] in ['LumiBased'] and 'events' in split_par:
```

Comment:
since `split_par` is a dict, the second condition should be
`(any("events" in x for x in split_par))`

Reply:
and try to run it locally to make sure